README.md

```diff
@@ -31,16 +33,25 @@ Then, under **Add a platform**, add a **Android app** or a **Apple app**. You ca
 Add your app **name** and **Bundle ID**. You can find your **Bundle Identifier** in the **General** tab for your app's primary target in XCode.
 
 #### Android steps
-Add your app's **name** and **package name**, Your package name is generally the **applicationId** in your app-level `build.gradle` file.
+Add your app's **name** and **package name**, Your package name is generally the **applicationId** in your app-level [build.gradle](https://github.com/appwrite/playground-for-flutter/blob/master/android/app/build.gradle#L41) file.
```

Comment:
maybe use a permalink? Because this points tosourceSets right now 
<img width="592" alt="image" src="https://github.com/appwrite/sdk-for-react-native/assets/1477010/e7bfe3d0-0f16-4253-af2a-9be74ae3da53">